### PR TITLE
feat: created nuxt3 blog example

### PR DESCRIPTION
**Pull request by Intevel**

### CODE REVIEW

File: get-started-building-a-nuxt-3-blog-with-directus/index.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+### What is Nuxt 3?
+
+Nuxt 3 is an open source framework making web development simple and powerful. With Vue 3 and experience hybrid rendering, powerful data fetching and new features. [Visit the offical website](https://v3.nuxtjs.org).
+
+## Setup 🚀
+
+### 1. Create a new Nuxt 3 project
+
+```bash
+npx nuxi init my-blog
```
Comment (rijkvanzanten) [inline get-started-building-a-nuxt-3-blog-with-directus/index.md:24]:
```suggestion
npx nuxt init my-blog
```

File: get-started-building-a-nuxt-3-blog-with-directus/index.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+cd my-blog/
+
+npm install
+```
+
+This created your Nuxt 3 application 💚.
+
+### 2. Create your Directus application [here](https://directus.cloud/login)
+
+### 3. Install the directus module & a CSS framework. We use TailwindCSS in this guide.
```
Comment (rijkvanzanten) [inline get-started-building-a-nuxt-3-blog-with-directus/index.md:35]:
```suggestion
### 3. Install the Directus module & a CSS framework. We use TailwindCSS in this guide.
```

File: get-started-building-a-nuxt-3-blog-with-directus/index.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+	modules: ["nuxt-directus", "@nuxtjs/tailwindcss"],
+	directus: {
+		url: "my-url",
+	},
+});
+```
+
+### Create a collection in the Directus admin dashboard 📄
+
+![image 1](1.webp)
```
Comment (rijkvanzanten) [inline get-started-building-a-nuxt-3-blog-with-directus/index.md:68]: Lets use proper alt tags and file names for these images. For both A11y and SEO reasons 🙂

File: get-started-building-a-nuxt-3-blog-with-directus/index.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+### Go to your collection and create example items.
+
+## You are ready to code! 💻
+
+### Open your Editor (In this Guide: Visual Studio Code) and go to the directory of the Nuxt project
+
+Delete the file `app.vue`, and create a new file `pages/index.vue`.
+
+**This is our new index page/home page.**
```
Comment (rijkvanzanten) [inline get-started-building-a-nuxt-3-blog-with-directus/index.md:98]:
```suggestion
**This is our new homepage.**
```

File: get-started-building-a-nuxt-3-blog-with-directus/index.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+
+Add the script to the page
+
+```ts
```
Comment (rijkvanzanten) [inline get-started-building-a-nuxt-3-blog-with-directus/index.md:102]: We either have to use html/vue, or remove the `<script>` part for this to be a valid code block

```suggestion
```html
```

File: get-started-building-a-nuxt-3-blog-with-directus/index.md [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+<script setup>
+const { getItems } = useDirectusItems();
+
+const posts = await getItems({ collection: "blog" });
+</script>f
```
Comment (rijkvanzanten) [inline get-started-building-a-nuxt-3-blog-with-directus/index.md:107]:
```suggestion
</script>
```

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]
Couple of little tweaks 🙂

File: get-started-building-a-nuxt-3-blog-with-directus/index.md (hunk 1 above)
Comment (Intevel) [reply get-started-building-a-nuxt-3-blog-with-directus/index.md:24]: No, Nuxi is the Nuxt 3 CLI.

File: get-started-building-a-nuxt-3-blog-with-directus/index.md [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+If you click on the title of an article you will be redirected to a dynamic page, which gets the id and fetches the corresponding article and renders it.
+
+![image 6](6.webp)
+
+**We are finished!** ✨
+
+We created a simple and minimalistic blog using Nuxt 3 and Directus. The design can be customized at any time, this guide should only show step-by-step how to build a blog. TailwindCSS did a lot of the design work for us, the nuxt-directus module gives us a quick and easy solution to access our Directus server.
+
+If you have questions feel free to create an Issue on github, or ask on the Nuxt Discord Server: https://discord.gg/zubPXvtspT
+You can find the code here: https://github.com/intevel/nuxt-directus/examples/nuxt3-directus-blog/
```
[the remark this answers is not in the document]
Comment (Intevel) [reply get-started-building-a-nuxt-3-blog-with-directus/index.md:172]: Yes sure.

File: get-started-building-a-nuxt-3-blog-with-directus/index.md (hunk 1 above)
Comment (rijkvanzanten) [reply get-started-building-a-nuxt-3-blog-with-directus/index.md:24]: Oh word? Haha nice. My b!

File: get-started-building-a-nuxt-3-blog-with-directus/index.md (hunk 3 above)
Comment (Intevel) [reply get-started-building-a-nuxt-3-blog-with-directus/index.md]: Fixed with new commit

**Comment by rijkvanzanten** [comment]
Alright! Sent you an invite to `directus/examples`. Lets put the `nuxt3` example in there, update the link in the guide, and this is good to go! ❤️ 

Thank you very much @Intevel ✨

**Comment by Intevel** [comment]
Thanks for that. I updated the link! 🌟

**Review by rijkvanzanten — APPROVED** [review]

**Comment by rijkvanzanten** [comment]
https://directus.io/guides/get-started-building-a-nuxt-3-blog-with-directus/ 🎉

**Comment by Intevel** [comment]
Thank you, it's not on the Guides page 😓

**Comment by rijkvanzanten** [comment]
It is! It's just at the bottom because the page is sorting the wrong way 🙈

**Comment by Intevel** [comment]
Yeah I see, maybe we should fix this. That's confusing 😂

**Comment by Intevel** [comment]
Is the website open-source? Maybe I can make a PR.

**Comment by rijkvanzanten** [comment]
> Yeah I see, maybe we should fix this. That's confusing 😂

Yeah! Just pushed up a fix 🙂

**Comment by rijkvanzanten** [comment]
> Is the website open-source? Maybe I can make a PR.

No, it's not (yet) 👍🏻

**Pull request merged**